docs/source/contributing.rst

```diff
@@ -54,3 +54,20 @@ To execute the test suite without the smoke tests:
 .. code-block:: none
 
 	$ python3 -m kordac.tests.start_tests --no_smoke
+
+Creating a release
+=======================================
+
+This is our current process for creating and publishing a Kordac release. This
+can only be performed by users that are part of the `uccser/Kordac team`, if you
```

Comment:
Why does this line have backticks around text? You could probably shorten `This can only be performed by users that are part of the `uccser/Kordac team`, if you are not on this team get in contact with someone who is.` to `This can only be performed by repository administrators`.